src/main/java/com/wcc/platform/domain/cms/pages/CollaboratorPage.java

```diff
@@ -8,4 +8,7 @@
 
 /** CMS Collaborators Page to highlight volunteers in the community. */
 public record CollaboratorPage(
-    @NotNull Page page, @NotNull Contact contact, @NotEmpty List<Member> collaborators) {}
```

Comment:
@amgc3 it is missing part of the implementation, where you have to add this attributes in the REST api: 

* AboutController: 
* CmsService: